person_capture/gui_app.py

```
@@ -3600,12 +3613,41 @@ def _set_lock(self, bbox_xywh: Tuple[int, int, int, int], frame_idx: int):
         self._lock_last_bbox = (x, y, w, h)
         self._lock_last_seen_idx = int(frame_idx)
 
+    def _set_lock_face_box(self, face_box_xyxy: Optional[Tuple[int, int, int, int]]) -> None:
+        if face_box_xyxy is None:
+            return
+        try:
+            x1, y1, x2, y2 = [int(round(v)) for v in face_box_xyxy]
```

Ruff (0.15.12)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)